known-issues.rst

```diff
@@ -53,6 +53,9 @@ Red Hat (and clones) specific
   Attempting to use MKL will cause the test suite to fail, automatically
   aborting the installation. :jirab:`DM-5105`
 
+- Ensure that the :command:`gcc` compiler version supports **C++14** as 
+  specified in the :ref:`Pre-requisites <source-install-redhat-prereqs>`.
```

Comment:
“Prerequisites” shouldn't be hyphenated.

Reply:
Also, this is not a “Red Hat (and clones) specific” issue: it's a requirement everywhere. Documenting “your system must fulfil the requirements for installation” as a “known issue” doesn't seem useful. :-)